org.idempiere.p2.targetplatform/org.idempiere.p2.targetplatform.target

```diff
@@ -1215,7 +1215,7 @@ Export-Package:        *;version="${version}";-noimport:=true
 version:               ${version_cleanup;${mvnVersion}}
 Bundle-SymbolicName:   wrapped.${mvnGroupId}.${mvnArtifactId}
 Bundle-Version:        ${version}
-Import-Package:        *;resolution:=optional
+Import-Package:        org.apache.commons.logging;version="[1.1.1,2)";resolution:=optional,org.apache.commons.logging.impl;version="[1.1.1,2)";resolution:=optional,org.apache.tools.ant;version="[1.7,2)";resolution:=optional,*;resolution:=optional
```

Comment:
isn't the commons logging version in current target platform 1.2 instead of 1.1.x ?